Scripts/aws_stack_provision.ps1

```diff
@@ -1,65 +1,149 @@
-## Powershell script for
+## Powershell script for provisioning AWS stack
 
 param (
 [parameter(Mandatory=$true)]    [string] $lansa_version,
-[parameter(Mandatory=$false)]   [string] $tag_lansa_version ## lansa version that would be used to tag cloned stack
+[parameter(Mandatory=$false)]   [string] $clone_lansa_version ## lansa version that would be used to create clone stack.
 
 )
 
-Write-Host "Searching for VM with Lansa Version tag = $lansa_version"
-
-$EXISTING_INSTANCE_COUNT = (((Get-EC2Instance -Filter @{ Name="tag:LansaVersion"; Values=$lansa_version }).Instances).InstanceId).count
-
-if ($EXISTING_INSTANCE_COUNT -eq 1 ){
-  
-	Write-Host "VM with Lansa Version tag = $lansa_version exist"
-	Write-Host "Checking VM Status"
-	$INSTANCE_ID = ((Get-EC2Instance -Filter @{ Name="tag:LansaVersion"; Values=$lansa_version }).Instances).InstanceId
-	$INSTANCE_STATE = (((Get-EC2InstanceStatus -IncludeAllInstance $true -InstanceId $INSTANCE_ID).InstanceState).Name).Value
+if ( [string]::IsNullOrWhiteSpace( $clone_lansa_version) )
+{
+   $clone_lansa_version = $lansa_version
+}
 
-  	if ($INSTANCE_STATE -eq "running"){
-		Write-Host "VM is already in running state"}
+$MYSQL_RDS_CFN_TEMPLATE_URL = "https://lansa-us-east-1.s3.amazonaws.com/db-regression-test/cloudformation_template/mysql_rds.cfn.template.yml"
```

Comment:
Even if we pass the template body in the command it will still store the template in S3, so I would recommend using template URL instead.

Reply:
But where is the template version controlled?
Thats the issue. I know all about CFN requiring an S3 location to create the stack. Just let it do that and keep the template in git.

Reply:
I have already pushed the same templated in the git as well So that way template version can be maintained and the same templates has been uploaded to S3 so that can be used as templateURL parameter.

Reply:
So we can proceed with current approach of TemplateURL ?

Reply:
No. -TemplateBody. They can get out of sync if they are in 2 places. Thats not good.